solution.cpp

```diff
@@ -1,23 +1,137 @@
-/* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+/* -------------------------------------------------------------------------*
+* Copyright &copy; 2015 Paul Sihavong paul.sihavong@hotmail.com				*
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)		*
+* ------------------------------------------------------------------------ */
 
-/**
- * TODO: program description
+/*Program Description*
+ *This program is meant to convert the user input of an integer from 1 to 4000 to a
+ *roman numeral. Assuming the user gives an input within a correct parameter, the program
+ *will convert the integer to the correct roman numeral.
  */
 
-// TODO: `#include`s and function prototypes
+#include <iostream>
+#include <string>
 
-// ----------------------------------------------------------------------------
+using std::cin;
+using std::cout;
+using std::endl;
+using std::string;
 
-int main() {
-    // TODO
+string romanNumeralConverter(int a, int c);
 
-    return 0;  // success
+int main() {
+	int a = 0; // variable to store user input
+	int c = 0; // variable to determine what roman numerals are used
+	char d = ' '; // char to determine if the user would like to use the calculator again
+	// the start of calculator processing
+	// each numeral that is used here (e.g. M, CM, D, etc) is for the shortest way to write a roman numeral
+	// we start at the biggest roman numeral factor because we want the biggest numeral in front
+	do{
+		cout << "Please enter a positive integer up to 4000: ";
+		cin >> a;
+		cout << endl;
+		cout << romanNumeralConverter(a, c) << endl;
+		cout << "Would you like to convert another integer? (Y/N) ";
+		cin >> d;
+		cout << endl;
+		a = 0, c = 0;   // if the user chooses to convert another numeral, the variables
+						// are initialized to 0 and "" to make sure the old number isn't shown
+		if (d == 'N' || d == 'n')
+			break;
+	} while (d != 'N' || d != 'n');
+	system("PAUSE");
+	return 0;  // success
 }
 
 // ----------------------------------------------------------------------------
 
-// TODO: other functions
-
+string romanNumeralConverter(int a, int c){
+	string b = "";
+	// *explanation of how each if statement segment works*
+	if (a / 1000 > 0){  // each if statement determines whether if the number is big enough to warrant use of a roman numeral
+						// e.g. 22 / 1000 > 0 fails and goes to the next if statement and the roman numeral M is not used
+		c = a / 1000;   // if the if statement does pass, the formula c = a / 1000 will determine how many M's we have to use
```

Comment:
since `c` is being used this way, it would have been better to make it a variable local to the function, rather than passing a value (which isn't used) from `main`